backends/tc/ebpfCodeGen.h

```diff
@@ -287,6 +289,11 @@ class EBPFControlPNA : public EBPF::EBPFControlPSA {
         BUG_CHECK(result != nullptr, "No register named %1%", name);
         return result;
     }
+    EBPFHashPNA *getHash(cstring name) const {
+        auto result = ::P4::get(pna_hashes, name);
+        BUG_CHECK(result != nullptr, "No hash named %1%", name);
```

Comment:
This function is always expected to a valid value?

Reply:
yes, removed bug check.